etc/scripts/influxdb-onboarding.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
This file is not used, it should be better removed.

Reply:
ok